client/src/jokes/Jokes.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import axios from 'axios';
+
+class Jokes extends React.Component {
+  state = {
+    jokes: []
+  }
+
+  componentDidMount() {
+    const endpoint = "http://localhost:3300/api/jokes"
+    const headers = { authorization: localStorage.getItem('jwt') }
+    axios
+      .get(endpoint, { headers })
+      .then(res => {
+        this.setState({
+          jokes: res.data
+        })
+      })
+      .catch(error => {
+        console.error(error);
```

Comment:
You could skip the extra step and use `console.error` as your callback function:
```javascript
  .catch(console.error);
```